lib/rules/media-feature-name-value-no-unknown/__tests__/index.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+const { messages, ruleName } = require('..');
+
+testRule({
+	ruleName,
+	config: [true],
+
+	accept: [
+		{
+			code: '@media screen and (min-width: 768px) {}',
+			description: 'Basic media feature',
+		},
+		{
+			code: '@media screen and ( min-resolution :  2dpcm ) {}',
+			description: 'A resolution',
+		},
+		{
+			code: '@media screen and (resolution: 10.1dpcm) {}',
+			description: 'A resolution with a floating point value',
+		},
+		{
+			code: '@media screen and (min-width: $sm) {}',
+			description: 'Non-standard syntax is not checked by this rule',
+		},
+		{
+			code: '@media (color) {}',
+			description: 'Boolean context',
+		},
+		{
+			code: '@media (color : 1) {}',
+			description: 'Integer value',
+		},
+		{
+			code: '@media screen and (min-width <= 768px) {}',
+			description: 'Range context, media feature in allowed list',
+		},
+	],
```

Comment:
Can we add `-webkit` and `-moz` (do we have other) tests?

Reply:
Do we want to ignore or validate any vendor prefixed media feature?

I don't have any data for the values of prefixed media features, so my initial reflex is to just ignore these.

I don't have a strong opinion on this.

----

Should be covered by tests either way :)

Reply:
@romainmenke Ideally - we should normalize them and validate, `no-vendor-prefixes` should catch them too, but I think it can be done in other PRs

Reply:
I've added normalization for prefixed idents in feature names en feature values.

Reply:
minimum: ignore by default
better: validate based on the value after the prefix
ideal: based on `media-feature-name-no-vendor-prefix` current config, validate using a list (non existing vendor-prefixed names should be detected)

Reply:
> ideal: based on media-feature-name-no-vendor-prefix current config, validate using a list (non existing vendor-prefixed names should be detected)

I don't think the config of other rules is available 🤔 
Making it available would also open the door for spooky action at a distance.

Vendor prefixes are also a bit of an obsolete issue.
Engines no longer roll out new features with a vendor prefix.
All existing prefixes will eventually die out.

Reply:
> Engines no longer roll out new features with a vendor prefix.

Do you have a source for this?

Reply:
https://developer.mozilla.org/en-US/docs/Glossary/Vendor_Prefix

Reply:
> So, how about continuing as is?

I started out by just ignored everything with a vendor prefix.
So a vendor prefixed media feature name with an invalid value type would fine, it would not give errors.

I changed this to stripping prefixes, so any vendor prefixed media feature name would be treated as if it were unprefixed.

Do I leave it as is, strip the prefixes, or do I revert to the original code?
I don't have a strong preference between these two. :)